python/tskit/trees.py

```diff
@@ -47,9 +49,14 @@
 from tskit import NULL
 
 
-CoalescenceRecord = collections.namedtuple(
-    "CoalescenceRecord", ["left", "right", "node", "children", "time", "population"]
-)
+@attr.s(slots=True, frozen=True, auto_attribs=True)
```

Comment:
No point in changing this as it's deprecated anyway.

Reply:
https://github.com/tskit-dev/tskit/pull/553/commits/9a743c46f8a9c965c983508196cd4e6526b35546